packages/main/src/TabContainer.ts

```diff
@@ -534,7 +506,7 @@ class TabContainer extends UI5Element {
 			return;
 		}
 
-		this._overflowItems = tabInstance.subTabs;
+		this._overflowItems = tabInstance.subTabs || [];
```

Comment:
Currently, Tab and TabSeparator implement the interface and TabSeparator doesn't have subTabs property. Because of that subTabs is marked as optional in the interface and typescript complains. Need to make one additional iteration on the interface declaration.

Reply:
Revise it again and found where the problem comes from. Could you please take a look if all the types in TabContainer are declared correctly because I have doubts how Tab and ITab are used.

Reply:
This is very possible. Can we check it after this gets merged?